src/Providers/Stripe/Customer/Service.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace PodPoint\Payments\Providers\Stripe\Customer;
+
+use PodPoint\Payments\Customer;
+use PodPoint\Payments\Token;
+use Stripe\Customer as StripeCustomer;
+use PodPoint\Payments\Providers\Stripe\Token as StripeToken;
+use PodPoint\Payments\Customer\Service as CustomerInterface;
+
+class Service implements CustomerInterface
+{
+    /**
+     * Tries to create a customer using the Stripe SDK.
```

Comment:
Can you please explain what is the reason for this change? as you contradict yourself
![Screenshot 2019-08-19 at 12 10 35](https://user-images.githubusercontent.com/53213064/63261183-b7629280-c27a-11e9-94c9-fc4f3b09e2cb.png)

Reply:
so I do! I think because no known exception is thrown from this method.

That's why it differs from the the payment service where a known exception could be thrown and therefore the language `Tries to...` is more appropriate.